javascript-conditionals/main.js

```diff
@@ -0,0 +1,118 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+/* isUnderFive Function */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+/* isEven Function */
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+/* startsWithJ Function */
+function startsWithJ(string) {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+/* isOldEnoughToDrink Function */
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+/* isOldEnoughToDrive Function */
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+/* isOldEnoughToDrinkAndDrive Function */
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age) {
+    return false;
+  }
+}
+
+/* categorizeAcidity Function */
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH <= 6 && pH >= 0) {
+    return 'acid';
+  } else if (pH >= 8 && pH <= 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+/* introduceWarnerBro Function */
+function introduceWarnerBro(name) {
```

Comment:
The function is supposed to **return** a string. Same goes for the following function :)